CRM/Member/Form/Membership.php

```diff
@@ -1363,12 +1361,11 @@ public function submit(): void {
       $params['source'] = $formValues['source'] ?: $params['contribution_source'];
       $params['trxn_id'] = $result['trxn_id'] ?? NULL;
       $params['is_test'] = $this->isTest();
-      if (!empty($formValues['send_receipt'])) {
+      $params['receipt_date'] = NULL;
+      if ($this->getSubmitValue('send_receipt') && $paymentStatus === 'Completed') {
+        // unset send-receipt option, since receipt will be sent when ipn is received.
```

Comment:
This comment seems a bit out of place now

Reply:
oh yeah - I've changed it to a 'don't do this'